snap/snapcraft.yaml

```diff
@@ -1,5 +1,5 @@
 name: minio
-version: git
+version: master
```

Comment:
I see i have created launchpad which seemed to fail in the same manner as you explained here.. https://launchpad.net/minio